### ISSUE TITLE
TriggerSetConfig is undefined until manually set by the user

### ISSUE THREAD

**Issue by Souma-Sumire** [quisquous/cactbot#5380]
## Plugin Version
OverlayPlugin.dll        0.19.18.0 
CactbotOverlay.dll       0.28.19.0 

## Reproduce
create a user.js
```
Options.Triggers.push({
  id: "test 20230504",
  zoneId: ZoneId.MatchAll,
  config: [
    {
      id: "test001",
      name: { en: "test001" },
      type: "select",
      options: { en: { a: "a", b: "b" } },
      default: "b",
    },
    {
      id: "test002",
      name: { en: "test002" },
      type: "select",
      options: { en: { a: "a", b: "b" } },
      default: "b",
    },
    {
      id: "test003",
      name: { en: "test003" },
      type: "checkbox",
      default: () => {
        return true;
      },
    },
    {
      id: "test004",
      name: { en: "test004" },
      type: "string",
      default: () => {
        return 'default string';
      },
    },
  ],
  triggers: [
    {
      id: "test triggerSetConfig 20230504",
      type: "GameLog",
      netRegex: NetRegexes.echo({ line: "test", capture: false }),
      run: (data, _matches) => {
        console.log("test", data.triggerSetConfig);
      },
    },
  ],
});

```
![image](https://user-images.githubusercontent.com/33572696/236146640-1c53e73d-62d3-4b7c-89dd-9d243432cf7e.png)

then i `/e test` to run trigger

print is {}
![image](https://user-images.githubusercontent.com/33572696/236144246-c8f1b91d-fe85-4133-b403-292c93473547.png)

After I manually set the option for 002 and reload raidboss

![image](https://user-images.githubusercontent.com/33572696/236144428-187432fa-616a-40d0-8119-5cb76bc0d95f.png)

![image](https://user-images.githubusercontent.com/33572696/236144508-6e860461-3c30-49a7-b52a-43f17d17bb07.png)

only test002 have value, other key is undefined.

**Comment by quisquous** [comment]
Thanks for the example.  I see at least one bug, so I'll try to get a fix in soon.

**Comment by quisquous** [comment]
@Souma-Sumire Do you have a local cactbot development set up, or do I need to do a release with #5381 before you can use this?

**Comment by Souma-Sumire** [comment]
> @Souma-Sumire Do you have a local cactbot development set up, or do I need to do a release with #5381 before you can use this?

Sorry, I don’t have a local development environment. I failed in step two of Steps to build plugin.

**Comment by quisquous** [comment]
> > @Souma-Sumire Do you have a local cactbot development set up, or do I need to do a release with #5381 before you can use this?
> 
> ~Sorry, I don’t have a local development environment.I don't know how to debug the config page.~
> 
> ~Edit: Maybe I got it. Wait a minute.~
> 
> Edit: Yes I think this issue has been fixed by #5381!

Ok, glad you got it sorted! I didn't see this so did a release anyway.  Let me know if there are any further issues.
